packages/cosmjs/src/utils/address.ts

```diff
@@ -76,27 +66,57 @@ export const getAddressHashFromPubKey = (compressedPublicKey: Uint8Array): Uint8
  * @param signature A byte array representing the digital signature.
  * @param pubKey A byte array representing the public key of the signer.
  * @returns True if the digest is verified to have been signed using the private key complementing the given public key.
+ * @category Utils
  */
 export const verifyDigest32 = (digest: Uint8Array, signature: Uint8Array, pubKey: Uint8Array): boolean => {
-  if (digest.length !== 32) {
-    throw new Error(`Invalid length of digest to verify: ${digest.length}`);
-  }
+	if (digest.length !== 32) {
+		throw new Error(`Invalid length of digest to verify: ${digest.length}`);
+	}
+
+	if (signature.length !== 64) {
+		throw new Error(`Invalid length of signature: ${signature.length}`);
+	}
 
-  if (signature.length !== 64) {
-    throw new Error(`Invalid length of signature: ${signature.length}`);
-  }
+	const secp256k1 = new EllipticCurve('secp256k1');
 
-  const secp256k1 = new EllipticCurve('secp256k1');
+	const r = signature.slice(0, 32);
+	const s = signature.slice(32);
 
-  const r = signature.slice(0, 32);
-  const s = signature.slice(32);
+	return secp256k1.verify(
+		digest,
+		{
+			r: Buffer.from(r).toString('hex'),
+			s: Buffer.from(s).toString('hex')
+		},
+		pubKeyToKeyPair(pubKey)
+	);
+};
+
+/**
+ * Returns a boolean representing if the given string is a valid Sei address.
+ * @param address The string we wish to verify.
+ * @returns True if a string is a valid Sei address
+ * @category Utils
+ */
+export const isValidSeiCosmosAddress = (address: string) => {
```

Comment:
i think we should keep the name `isValidSeiAddress` and use "sei address" and "evm address" when referring to bech32 vs 0x